conflict.txt

```diff
@@ -1 +1,14 @@
-This is the text after resolved conflict.
+What is 'git stash' and how to use it?
+- Stash is used to avoid commiting of incomplete implementation in the moment when you want to switch to another branch.
+- I do some coding. Stage > Stash, Switch to another branch. But you have your stashed changes available at any time.
+
+What is 'git reset' and when we shouldn't use it?
+- Use git reset when you want to revert changes made in particular branch. Reverting files from index while you can keep them in working directory.
+- We probably should not use it when we are afraid of losing our local changes.
```

Comment:
we shouldn't use it for files been already pushed to remote repository since somebody could make some changes based on our code and it will look like some old code disappeared